surf/src/vcs/git.rs

```diff
@@ -121,28 +121,29 @@ impl From<git2::Buf> for Signature {
     }
 }
 
-/// Select the branches to be listed.
-pub enum BranchSelector {
+/// Determines whether to look for local or remote references or both.
+pub enum RefScope {
```

Comment:
Hmmmm maybe `RefScope` should now take the category it's interested in, i.e. `head`, `tags`, `notes`, etc. What do you think?

I say this because the `From` impl defaults to `heads`, which I think you've mentioned may be a problem

Reply:
The `From` implementation that hard-codes `heads` is indeed a problem. But adding the category as a fieldseems to add some redundancy. `RefScope` is passed to `list_branches()` and `list_tags()` which already determines the category.

To address the issue we could remove the `From` instance and move that code to it’s [single call-site in upstream](https://github.com/radicle-dev/radicle-upstream/blob/8a6413549f9bd37bbaafbb77eb3de09853e83955/proxy/api/src/http/source.rs#L165).

Reply:
Ya that probably makes the most sense, good shout :+1: